core/src/main/java/com/graphhopper/GraphHopper.java

```diff
@@ -620,7 +620,7 @@ private static ElevationProvider createElevationProvider(GraphHopperConfig ghCon
             elevationProvider.setBaseURL(baseURL);
         elevationProvider.setDAType(elevationDAType);
         return elevationProvider;
-    }
+    } */
```

Comment:
Can you explain why we need to remove ElevationProvider here? It is just an interface and should not make a problem or does it? Couldn't you set ElevationProvider.NOOP?

The advantage would be that the CI pipeline would not fail.

Reply:
I only had to make sure that HeightTile.java is not needed, since it contains references to java.awt.*, which are not supported. Let me try to rewrite it for this usecase.

Reply:
Ok, but now the elevation specific tests fail. For a reason . How to proceed from here ?
Should we comment the tests as well ?
Since we do not need Elevations at all on mobile ... We will never generate the graph on mobile.

Reply:
Hmmh, tricky. But we could make it working when we move the whole init method out of the GraphHopper class. Probalby this is a good idea in general, but to avoid a delay here we could for now exclude building the web module in this branch. Will try.

BTW: our usual workflow is to keep all the commits of a pull request (instead of overwriting old commits) and then squash merge them as one commit into master. So the master branch stays relatively "clean" and the pull request branch still has everything in it and we can see how it evolved.

Reply:
Can you try the following change:

```patch
--- a/.travis.yml
+++ b/.travis.yml
@@ -38,7 +38,7 @@ before_script:
   - cd web && npm install && BROWSERIFYSWAP_ENV='development' npm run bundleProduction && cd ..
 
 script:
-  - "mvn -Dkey=$API_KEY clean install -B"
+  - "mvn -Dkey=$API_KEY clean install -B -pl \!web"
```

With that the branch is red only if something is broken.